static/src/js/components/Map/__tests__/ShapefileLayerExtended.test.js

```diff
@@ -0,0 +1,121 @@
+import { ShapefileLayerExtended } from '../ShapefileLayerExtended'
+
+describe('ShapefileLayerExtended', () => {
+  describe('separateMultiPolygons', () => {
+    let layer
+
+    beforeEach(() => {
+      layer = new ShapefileLayerExtended({})
+    })
+
+    it('should separate MultiPolygon features into individual Polygon features', () => {
+      const geojson = {
+        type: 'FeatureCollection',
+        features: [
+          {
+            type: 'Feature',
+            properties: { name: 'Region Name' },
+            geometry: {
+              type: 'MultiPolygon',
+              coordinates: [
+                [[[0, 0], [1, 1], [1, 0], [0, 0]]],
+                [[[2, 2], [3, 3], [3, 2], [2, 2]]]
+              ]
+            }
+          }
+        ]
+      }
+
+      layer.separateMultiPolygons(geojson)
+
+      expect(geojson.features.length).toBe(2)
+      expect(geojson.features[0].geometry.type).toBe('Polygon')
+      expect(geojson.features[0].geometry.coordinates).toEqual([[[0, 0], [1, 1], [1, 0], [0, 0]]])
+      expect(geojson.features[0].properties).toEqual({ name: 'Region Name' })
+      expect(geojson.features[1].geometry.type).toBe('Polygon')
+      expect(geojson.features[1].geometry.coordinates).toEqual([[[2, 2], [3, 3], [3, 2], [2, 2]]])
+      expect(geojson.features[1].properties).toEqual({ name: 'Region Name' })
+    })
+
+    it('should handle GeometryCollection with MultiPolygon features', () => {
```

Comment:
```suggestion
    test('should handle GeometryCollection with MultiPolygon features', () => {
```
nit; its just an alias but, we use test throughout the rest (except in playwright)